docs/source/guides/network_node/ursula_configuration_guide.rst

```diff
@@ -17,6 +17,36 @@ The worker's ethereum account must remain unlocked to send automated work confir
 ether to pay for transaction gas; however, it is *not* necessary (and potentially risky) to hold NU tokens on a worker's
 account for any reason.
 
+
+Worker Requirements
+-------------------
+
+Aside from the :ref:`base requirements <base-requirements>` for installation of the ``nucypher`` library:
+
+* 4GB RAM (minimum)
+* 2 CPUs (minimum)
+* 256GB HDD
+* TCP Port 9151 opened for network communication
+* Access to a fully synced Ethereum web3 provider e.g. local node, infura, alchemy etc. (see :ref:`using-eth-node`) is
+  required to read and write to smart contracts
+
+In order to be a successful Ursula operator, you will need a machine (physical or virtual) which
+can be kept online and ready to perform cryptographic operations for the network. However, short
+temporary service disruptions such as upgrades are understandable. Common cloud setups used
+include `AWS T3.medium <https://aws.amazon.com/ec2/instance-types/t3/>`_ and
+`Digital Ocean 4GB Basic Droplet <https://www.digitalocean.com/pricing/>`_.
```

Comment:
```suggestion
temporary service disruptions such as upgrades are understandable. Cloud setups are acceptable – for example,`Digital Ocean 4GB Basic Droplet <https://www.digitalocean.com/pricing/>`_ satisfies the memory and processing power requirements listed above.
```

We can't stop stakers using AWS (nor should we), but I reckon we should stop short at officially recommending the archetypal 'merchant of infrastructure' – this slogan is on our DAO page! 😇

Reply:
If we're going to emphasize AWS :-1: Let's give some extra attention to what it means to run a node at home.  ( dappnode, desktops, lightweight servers, NUC, RPI, etc.)